### ISSUE TITLE
Add a cargo for trucks

### ISSUE THREAD

**Issue by emsici** [traccar/traccar-web#808]
**Is your feature request related to a problem? Please describe.**
Yes, for delivery companies and trucks. Every trucks has more than one package that needs to be send.

**Describe the solution you'd like**
A dropdown input based on identifier, so, every identifier has more locations. If he checks a locations, the locations dissapear from list. Something like to-go list. He choses the order.

**Describe alternatives you've considered**
This is the simpliest one.

**Additional context**
Truck A has clients 1,2,3,4 and 5. On the client android app he will select where he goes, after that he starts the service. After the first client he goes to the second one repeating the start-stop step.

**Comment by dkyeremeh** [comment]
I think this falls outside the scope of Traccar. Traccar is meant to be used to find the location of your vehicle. 

It is not intended to be used to plan trips and delivery

**Comment by tananaev** [comment]
Yes, I would say it's out of scope, at least for now. But I would like to keep it open in case we want to implement it as some kind of extension later. It's s pretty common request.
